web/impact/impact/v1/views/cancel_office_hour_session_view.py

```diff
@@ -0,0 +1,126 @@
+from pytz import timezone
+
+from django.conf import settings
+from django.contrib.auth import get_user_model
+from django.template import loader
+
+from rest_framework.exceptions import PermissionDenied
+from rest_framework.generics import get_object_or_404
+from rest_framework.response import Response
+
+from accelerator.models import MentorProgramOfficeHour
+from impact.minimal_email_handler import MinimalEmailHandler
+from impact.permissions.v1_api_permissions import OfficeHourPermission
+from impact.v1.views.impact_view import ImpactView
+
+User = get_user_model()
+
+DEFAULT_TIMEZONE = 'UTC'
+SUBJECT = '[Office Hours] Canceled: {date}, {start_time}'
+
+STAFF_NOTIFICATION = 'Canceled office hours on behalf of ' \
```

Comment:
Not super important, but house style typically uses parens and not trailing backslashes to concatenate strings across line breaks.  
```
STAFF_NOTIFICATION = ('Canceled office hours on behalf of ' 
                     '{mentor_name} at {start_time} - {end_time} on {date}')
```

For consistency, let's stick with that.